dapr-1.15.yaml

```diff
@@ -0,0 +1,290 @@
+package:
+  name: dapr-1.15
+  version: "1.15.3"
+  epoch: 0
+  description: Portable, event-driven, runtime for building distributed applications across cloud and edge.
+  dependencies:
+    provides:
+      - dapr=${{package.full-version}}
+  copyright:
+    - license: Apache-2.0
+
+var-transforms:
+  - from: ${{package.version}}
+    match: ^(\d+\.\d+)\.\d+$
+    replace: "$1"
+    to: major-minor-version
+
+environment:
+  contents:
+    packages:
+      - protobuf
+      - protobuf-dev
+      - protoc
+      - protoc-gen-go
+      - protoc-gen-go-grpc
+  environment:
+    GO_TAGS: "allcomponents"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: c3687714f9592def48b14b8300bf5170fe8a1bff
+      repository: https://github.com/dapr/dapr
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/golang-jwt/jwt/v5@v5.2.2
+        golang.org/x/oauth2@v0.27.0
+        golang.org/x/net@v0.36.0
+        github.com/choleraehyq/pid@v0.0.20
+        github.com/golang-jwt/jwt/v4@v4.5.2
+
+  - runs: |
+      # Generate proto code
+      sed -i '/check-proto-version:/,/^$/d' Makefile
+      make gen-proto
+
+subpackages:
```

Comment:
The subpackages does seem a bit duplicated, was wondering if we could use `items:` iteration here.

Reply:
Good idea, let me try that real quick, thanks Furkan